React/Modules/RCTTiming.m

```diff
@@ -133,6 +138,14 @@ - (dispatch_queue_t)methodQueue
   return RCTJSThread;
 }
 
+- (NSDictionary *)constantsToExport
```

Comment:
We should add comments on both sides saying to keep the constants in sync across JS and Obj-C then